sopel/modules/currency.py

```diff
@@ -4,43 +4,43 @@
 # Licensed under the Eiffel Forum License 2
 from __future__ import unicode_literals, absolute_import, print_function, division
 
-import time
 import re
 import requests
+import time
 
-from sopel.module import commands, example, NOLIMIT, rule
-from sopel.tools import stderr
 from sopel.config.types import StaticSection, ValidatedAttribute
+from sopel.logger import get_logger
+from sopel.module import commands, example, NOLIMIT, rule
 
-rates_fiat_json = {}
-rates_btc_json = {}
-
-
-fiat_url = 'https://api.exchangeratesapi.io/latest?base=EUR'
-fixer_url = 'http://data.fixer.io/api/latest?base=EUR&access_key={}'
-crypto_url = 'https://apiv2.bitcoinaverage.com/indices/global/ticker/short?crypto=BTC'
-regex = re.compile(r'''
+FIAT_URL = 'https://api.exchangeratesapi.io/latest?base=EUR'
```

Comment:
Two blank lines before constants (relatively new convention, standardized to the best of my ability in #1402).